### PR TITLE
[zh-cn]: Correct the link in tutorial of command line

### PR DESCRIPTION
**Pull request by kino14910**

### CODE REVIEW

File: files/zh-cn/learn/tools_and_testing/understanding_client-side_tools/command_line/index.md [hunk 1]
```diff
@@ -261,9 +261,9 @@ ls | wc -l
 
 ## 一个稍微复杂一点的例子
 
-让我们看一些更复杂的东西。我们将首先尝试获取 MDN 的“获取”页面的内容 `curl` 命令 (可用于从 url 请求内容)[https://developer.mozilla.org/en-US/docs/Web/API/WindowOrWorkerGlobalScope/fetch](/zh-CN/docs/Web/API/WindowOrWorkerGlobalScope/fetch).
+让我们看一些更复杂的东西。我们将首先尝试获取 MDN 的“获取”页面的内容 `curl` 命令 (可用于从 url 请求内容)[https://developer.mozilla.org/zh-CN/docs/Web/API/WindowOrWorkerGlobalScope/fetch](/zh-CN/docs/Web/API/WindowOrWorkerGlobalScope/fetch).
```
Comment (JasonLamv-t) [inline files/zh-cn/learn/tools_and_testing/understanding_client-side_tools/command_line/index.md:264]:
```suggestion
让我们看一些更复杂的东西。

我们将首先尝试使用 `curl` 命令（可用于从 url 请求内容）从 [https://developer.mozilla.org/zh-CN/docs/Web/API/WindowOrWorkerGlobalScope/fetch](/zh-CN/docs/Web/API/WindowOrWorkerGlobalScope/fetch) 获取 MDN 的“fetch”页面的内容 .
```

**Review by JasonLamv-t — CHANGES_REQUESTED** [review]

File: files/zh-cn/learn/tools_and_testing/understanding_client-side_tools/command_line/index.md [hunk 2]
```diff
@@ -261,9 +261,9 @@ ls | wc -l
 
 ## 一个稍微复杂一点的例子
 
-让我们看一些更复杂的东西。我们将首先尝试获取 MDN 的“获取”页面的内容 `curl` 命令 (可用于从 url 请求内容)[https://developer.mozilla.org/en-US/docs/Web/API/fetch](/zh-CN/docs/Web/API/fetch).
+让我们看一些更复杂的东西。我们将首先尝试获取 MDN 的“获取”页面的内容 `curl` 命令 (可用于从 url 请求内容)[https://developer.mozilla.org/zh-CN/docs/Web/API/WindowOrWorkerGlobalScope/fetch](/zh-CN/docs/Web/API/WindowOrWorkerGlobalScope/fetch).
 
-但是，这个 URL 是页面的旧位置。如果你在一个新的浏览器标签中输入它，你将 (最终) 被重定向到[https://developer.mozilla.org/en-US/docs/Web/API/fetch](/zh-CN/docs/Web/API/fetch).
+但是，这个 URL 是页面的旧位置。如果你在一个新的浏览器标签中输入它，你将 (最终) 被重定向到[https://developer.mozilla.org/zh-CN/docs/Web/API/fetch](/zh-CN/docs/Web/API/fetch).
```
Comment (yin1999) [inline files/zh-cn/learn/tools_and_testing/understanding_client-side_tools/command_line/index.md:266]:
```suggestion
但是，这个 URL 是页面的旧位置。如果你在一个新的浏览器标签中输入它，你将 (最终) 被重定向到 [https://developer.mozilla.org/zh-CN/docs/Web/API/fetch](/zh-CN/docs/Web/API/fetch)。
```

File: files/zh-cn/learn/tools_and_testing/understanding_client-side_tools/command_line/index.md [hunk 3]
```diff
@@ -284,7 +286,7 @@ curl https://developer.mozilla.org/docs/Web/API/fetch -L -I | grep location
 输出应该是这样的 (`curl` 首先会输出一些下载计数器之类的东西):
 
 ```bash
-location: /en-US/docs/Web/API/fetch
+location: /zh-CN/docs/Web/API/fetch
```
Comment (yin1999) [inline files/zh-cn/learn/tools_and_testing/understanding_client-side_tools/command_line/index.md:289]: The location given in the example is the actual behavior of the server:

image

**Review by yin1999 — CHANGES_REQUESTED** [review]
In [previous updates to url's local](https://github.com/mdn/translated-content/pull/14273), we specifically skipped this section in the doc because it would make the context of the document inconsistent.

The English document has reorganized this section. I think we should completely synchronize the content instead of just modifying a few links.

Edit:

Left a `-1` here to avoid introducing incorrect content

**Pull request closed without merging**